.github/workflows/i18n.yml

```diff
@@ -1,63 +1,46 @@
-name: Locale Updater
+name: Update locale files
 
 on:
   push:
     branches:
-      - master
       - main
+      - master
+
+permissions:
+  pull-requests: write
+  contents: write
```

Comment:
Why add this permission?

Reply:
it's required to [write content](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs), in this case the locale files we're updating.